packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js

```diff
@@ -77,6 +77,10 @@ class EnumerationWidget extends React.Component {
 		};
 	}
 
+	componentWillReceiveProps(nextProps) {
+		this.setState({ ...this.state, items: nextProps.formData });
```

Comment:
Indeed, the formData is already well formatted and this transformation is unnecessary. I remove it.